js/app-user.js

```diff
@@ -80,9 +80,9 @@ class User {
     this.resetUser();
   }
 
-  resetUser() {
+  resetUser(justLoggedOut) {
     this.username = undefined;
```

Comment:
It takes a few seconds to find out if the current user is logged in or not. We shouldn't assume the current user is anonymous by setting `user.loggedin` to `false`.